src/Validation.Common/PackageValidationAuditor.cs

```
@@ -98,17 +98,31 @@ public PackageValidationAuditor(CloudStorageAccount cloudStorageAccount, string
                 packageId, 
                 packageVersion);
 
-            await StoreAuditAsync(validationId, packageId, packageVersion, 
-                packageValidationAudit =>
-                {
-                    packageValidationAudit.Completed = completed;
-                    return packageValidationAudit;
-                });
+            try
+            {
+                await StoreAuditAsync(validationId, packageId, packageVersion, 
+                    packageValidationAudit =>
+                    {
+                        packageValidationAudit.Completed = completed;
+                        return packageValidationAudit;
+                    });
+            }
+            catch (Exception e)
```

the previous behavior was to throw the exception,  and here we swallow it. What is the impact of this behavior change?